modules/logbucket/variables.tf

```diff
@@ -46,3 +46,21 @@ variable "grant_write_permission_on_bkt" {
   type        = bool
   default     = true
 }
+
+variable "enable_analytics" {
+  description = "(Optional) Whether or not Log Analytics is enabled. A Log bucket with Log Analytics enabled can be queried in the Log Analytics page using SQL queries. Cannot be disabled once enabled."
+  type        = bool
+  default     = false
+}
+
+variable "link_analytics_dataset" {
```

Comment:
@daniel-cit instead of using two separate variable `enable_analytics` and `linked_dataset_id` can we just use one variable `linked_dataset_id`? Set default value to null and if a value is passed then create resource `google_logging_linked_dataset` otherwise ignore?

Reply:
done